scripts/Muon/GUI/Common/results_tab_widget/results_tab_model.py

```diff
@@ -103,11 +117,7 @@ def log_selection(self, existing_selection):
         format matches that of the ListSelectorPresenter class' model.
         """
         selection = {}
-        fits = self._fit_context.fit_list
-        if not fits:
-            return selection
-
-        logs = log_names(fits[0].input_workspace)
+        logs = self._fit_context.log_names(filter_fn=_log_should_be_displayed)
```

Comment:
isn't this a function and therefore needs brackets?

Reply:
It's pass the function to `log_names` for that to call it so we don't need the brackets here.